linkatos/activities.py

```diff
@@ -39,10 +39,9 @@ def event_consumer(url_cache_list, slack_client, bot_id,
 
         if event['type'] == 'reaction_added' and len(url_cache_list) > 0:
             reaction = parser.parse_reaction_added(event)
-            index = react.is_confirmation(reaction['reaction'], url_cache_list,
-                                          reaction['to_id'])
+            index = react.get_index(url_cache_list, reaction['to_id'])
 
-            if react.is_expected_reaction(index):
+            if react.is_expected_reaction(index, reaction['reaction']):
```

Comment:
my problem with that is that the index is used to check if is expected but also to remove, add, etc.